wire-ios-link-preview/WireLinkPreviewTests/TestPlans/E2ETests.xctestplan

```diff
@@ -0,0 +1,159 @@
+{
+  "configurations" : [
+    {
+      "id" : "11F39FD2-5EA4-4A6F-BE05-030254B4AD6B",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "language" : "en",
+    "region" : "DE",
+    "testExecutionOrdering" : "random"
+  },
+  "testTargets" : [
+    {
+      "skippedTests" : [
+        "ImageDownloaderTests",
```

Comment:
This is a copy of the `AllTests` but all tests being disabled and only E2ETests enabled.